### DISCUSSION TITLE
Add constant UTC offset time zones to dashboard time zone selector

### DISCUSSION THREAD

**Discussion by sandermirme** [grafana/grafana#38216] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Add constant UTC offset time zones to the dashboard time zone selector. E.g. UTC+1, UTC+2, etc.

**Why is this needed**:
In certain cases (e.g. scientific environmental monitoring) a year-round constant utc offset time zone is used to represent data to avoid ambiguities due to daylight saving jumps. Almost all of the country based time zones that are currenty available in Grafana have either ongoing or past daylight savings periods or other perculiarities / discontinuities at some points.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
